src/commoncode/codec.py

```diff
@@ -62,7 +62,7 @@ def to_base_n(num, base):
 def to_base85(num):
     """
     Convert `num` number to a string representing this number in base 85,
-    padded as needed.
+    padding is done as needed.
```

Comment:
Are you sure this is a valuable comment update?

Reply:
Now realizing that this isn't of much value. Reverting this to as it was.

Reply:
> @PyAgni Thank you for the update. Do you mind to squash these two commits in a single one?

Done. Hope everything seems good now.